[CZBANK-33] fix: avatar not changing, fixed double stringify

silviczka opened the pull request:
The Avatar picture in User Profile was not changing when clicked on btn Generate new avatar due to double use of stringify, once in "user.service.ts" and then in "user-repository.ts", which is fixed now

silviczka committed:
fix: avatar not changing, fixed double stringify

silviczka commented:
https://czechitas-jira.atlassian.net/browse/CZBANK-33

postulkam reviewed (dismissed):
Prosim o odstraneni zmeny v `.env.example`.

Na kod davam Approve, hezkys e ti podarilo najit chyby. :) Takze pak muzes udelat Squash (klidne se ozvi a pustime se do toho spolu :) )

vojtech-cerveny committed:
chore: add database setup script and update package.json

- Introduced a new script `setup-database.ts` for setting up the database, including generating the Prisma client, running migrations, and seeding the database if applicable.
- Updated `package.json` to include a new command `db:ci:setup` for executing the database setup script.

silviczka committed:
chore: return erased file

postulkam reviewed (approved):
LGTM
